desktop/lib/messagebundles/default.po

```diff
@@ -4269,6 +4269,25 @@ msgstr ""
 msgid "Enable Distributed Hash Table (DHT)"
 msgstr ""
 
+#: /Users/gubatron/workspace.frostwire/frostwire/desktop/src/com/limegroup/gnutella/gui/options/panes/TorrentConnectionPaneItem.java:42
```

Comment:
the translation files are all generated from the `I18n.tr("some string")` code calls all over the project. Using `gradle gettextXXXX` tasks.

```
gettextExtract -> extracts all the I18n.tr() strings, into the default file
gettextBundle -> grabs all the other .po files and creates a messages.jar file
```

We only touch the other language files, not the default (english one)